src/plugins/orangehrmBuzzPlugin/config/routes.yaml

```diff
@@ -43,6 +43,13 @@ apiv2_buzz_comment:
   requirements:
     shareId: '\d+'
 
+apiv2_buzz_url_validation:
+  path: /api/v2/buzz/shares/url
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ POST ]
```

Comment:
refer 
`src/plugins/orangehrmPimPlugin/Api/ValidationEmployeeOtherEmailAPI.php`
and the route. and use GET method